docs/install-mender.md

```diff
@@ -122,22 +123,47 @@ This method requires appending `local.conf` with necessary recipes by using the
 
 ```
 IMAGE_INSTALL:append = " \ 
-    thin-edge \
     tedge \ 
-    tedge-mapper \
-    tedge-agent \
-    c8y-configuration-plugin \
-    c8y-log-plugin \
-    tedge-apt-plugin \
-    tedge-dummy-plugin \
-    tedge-watchdog \
+    tedge-bootstrap \
+    tedge-inventory \
     tedge-state-scripts \
+    tedge-firmware
 "
 ```
-Then you can use any image delivered by the Yocto team. Thin-edge requires at least `core-image-minimal` to function correctly:
+
+You must set fixed uid/gid to avoid permissions problems on the `/data` partition. You can do that by inheriting the `extrausers` class in your image recipe or the `local.conf` file:
+
+```
+inherit extrausers
+EXTRA_USERS_PARAMS = "\
+    groupmod -g 960 mosquitto; \
+    usermod -u 961 mosquitto; \
+"
+```
+
+We recommend switching to `Network Manager`. Append your `local.conf` with the following recipes:
```

Comment:
question: why do we recommend using NetworkManager?

Reply:
The main reason for recommending NetworkManager is that the mender state scripts which copy data from the current partition to the new one supports copying the NetworkManager configuration files (e.g. wifi connection info), so that you don't lose your wifi connection during an update. Using another network manager would be possible but the logic to persist the appropriate files would have to added to the mender state scripts.